From c19af19ab400ebdd137d785f47018f423433b375 Mon Sep 17 00:00:00 2001
From: Zoltán Böszörményi <zboszor@gmail.com>
Date: Thu, 25 Aug 2022 10:25:15 +0000
Subject: [PATCH 1/5] Don't use -Werror for caffe2 files
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Pooling.cpp has some unused variables and the compiler chokes
on them. Remove -Werror for that specific file to avoid the
compiler error.

Signed-off-by: Zoltán Böszörményi <zboszor@gmail.com>
Upstream-Status: Inappropriate [Yocto specific]
---
 caffe2/CMakeLists.txt | 2 --
 1 file changed, 2 deletions(-)

diff --git a/caffe2/CMakeLists.txt b/caffe2/CMakeLists.txt
index 328210c9782..4cde8f3235d 100644
--- a/caffe2/CMakeLists.txt
+++ b/caffe2/CMakeLists.txt
@@ -536,8 +536,6 @@
 # Required workaround for LLVM 9 includes.
 if(NOT MSVC)
   set_source_files_properties(${TORCH_SRC_DIR}/csrc/jit/tensorexpr/llvm_jit.cpp PROPERTIES COMPILE_FLAGS -Wno-noexcept-type)
-  # Force -Werror on several files
-  set_source_files_properties(${CMAKE_CURRENT_LIST_DIR}/../aten/src/ATen/native/mkldnn/Pooling.cpp PROPERTIES COMPILE_FLAGS "-Werror")
 endif()
 # Disable certain warnings for GCC-9.X
 if(CMAKE_COMPILER_IS_GNUCXX AND (CMAKE_CXX_COMPILER_VERSION VERSION_GREATER 9.0.0))
-- 
2.31.1

